templates/contest/list.html

```diff
@@ -228,7 +236,9 @@ <h4>{{ _('Upcoming Contests') }}</h4>
             <table class="contest-list table striped">
                 <thead>
                 <tr>
-                    <th>{{ _('Contest') }}</th>
+                    <th style="width:90%">
```

Comment:
Is this supposed to be `width:90%`?

Reply:
I originally did that to maintain consistency with the other contest groups (past, active, and current), but you are correct that it shouldn't be `width:90%` because upcoming contests don't have user counts or join buttons yet.